txpipe/utils/nmt_utils.py

```diff
@@ -72,14 +71,14 @@ def apply_window(self, b, c_ell):
         return (c_ell[ell] * weight).sum() / weight.sum()
 
     @classmethod
-    def from_binning_info(cls, ell_min, ell_max, n_ell, ell_spacing):
+    def from_binning_info(self, ell_min, ell_max, n_ell, ell_spacing):
```

Comment:
This shouldn't be `self`, it should be `cls`.  That's because `@classmethod` makes it so this method isn't attached to a specific instance of the class, but the class itself.  So the first argument isn't an instance of `MyNmtBinFlat`, it's the class object `MyNmtBinFlat` itself.  That means that we can only call other class methods from this method.  And the standard is to make this clear by calling the first arg `cls`.

More info: https://stackoverflow.com/questions/12179271/meaning-of-classmethod-and-staticmethod-for-beginner